src/applications/search/actions/index.js

```diff
@@ -41,11 +41,11 @@ export function fetchSearchResults(query, page, options) {
           meta: response.meta,
         });
       })
-      .catch(error =>
+      .catch(error => {
         dispatch({
           type: FETCH_SEARCH_RESULTS_FAILURE,
-          errors: error.errors,
-        }),
-      );
+          errors: [error],
```

Comment:
Out of curiosity, other than showing the UI component, do y'all log the error shown?

Reply:
great catch! all of this is legacy so now would be an excellent time to add some sentry logging